test/checkout.int.t.sol

```diff
@@ -991,12 +995,13 @@ contract Checkout_buyingWithCustomFee is Checkout {
       owner,
       owner,
       certificateAmount,
-      customFee
+      customFee,
+      customPriceMultiple
     );
     vm.stopPrank();
 
     Vm.Log[] memory entries = vm.getRecordedLogs();
-    uint256 createCertificateEventIndex = 6;
+    uint256 createCertificateEventIndex = 8;
```

Comment:
the `Approval` event note above is why this index changed when I approved a specific amount instead of MAX_INT, both here and in the other test

Reply:
that's interesting. We should track that behavior in a test somewhere. (not for this pr)